sources/packages/web/src/views/supporting-user/SupportingInformation.vue

```diff
@@ -56,21 +54,66 @@
       :data="initialData"
       :readOnly="submitting"
       @submitted="submitted"
+      @loaded="formLoaded"
     ></formio>
+    <v-row v-if="showNav">
+      <v-col>
+        <v-btn
+          color="primary"
+          v-show="!isFirstPage"
+          variant="outlined"
+          data-cy="previousSection"
+          @click="wizardGoPrevious"
+          >Previous section</v-btn
+        >
+      </v-col>
+      <v-col>
+        <v-btn
+          class="float-right"
+          color="primary"
+          v-show="!isLastPage"
+          @click="wizardGoNext"
+          >Next section</v-btn
+        >
+        <v-btn
+          class="float-right"
+          :disabled="!isLastPage || submitting"
+          v-show="!isFirstPage"
+          color="primary"
+          @click="wizardSubmit()"
+        >
+          <v-progress-circular
```

Comment:
Vuetify now has the progress property `loading`. This `v-progress-circular` can be removed.

Reply:
ok. Thanks!

Reply:
Was it changed, I am not able to see the file changed.

Reply:
yes, please check it again.